src/pages/en/reference/adapter-reference.md

```diff
@@ -156,3 +156,31 @@ if(app.match(request)) {
 ```
 
 You can usually call `app.render(request)` without using `.match` because Astro handles 404s if you provide a `404.astro` file. Use `app.match(request)` if you want to handle 404s in a different way.
+
+## Allow installation via `astro add`
+
+[The `astro add` command](/en/reference/cli-reference/#astro-add) allows users to easily add integrations and adapters to their project. If you want _your_ adapter to be installable with this tool, **add `astro-adapter` to the `keywords` field in your `package.json`**:
+
+```json
+{
+  "name": "myadapter",
```

Comment:
Note: I added a better explanation to this (and the others) review comments I did below, I'm not sure if GitHub adds my comments in the order I added them 😅 
```suggestion
  "name": "my-adapter",
```